release_docs/INSTALL_CMake.txt

```diff
@@ -46,36 +50,33 @@ Note:
 II. Quick Step Building HDF5 Libraries with CMake Script Mode
 ========================================================================
 This short set of instructions is written for users who want to quickly
-build the HDF5 C and C++ static libraries and tools with SZIP and ZLIB
-compression included.  This procedure will use the default settings in
-the config/cmake/cacheinit.cmake file.
+build the HDF5 C, C++ and Fortran shared libraries and tools
+from the HDF5 source code package using the CMake tools. This procedure
+will use the default settings in the config/cmake/cacheinit.cmake file.
 HDF Group recommends using the ctest script mode to build HDF5.
 
-The following files referenced below are available at the HDF web site:
-     https://portal.hdfgroup.org/display/support/Building+HDF5+with+CMake
-
-Single compressed file with all the files needed, including source:
-    CMake-hdf5-1.15.X.zip or CMake-hdf5-1.15.X.tar.gz
-
-Individual files included in the above mentioned compressed files
------------------------------------------------
-CMake build script:
+-------------------------------------------------------------------------
+-------------------------------------------------------------------------
+Individual files needed as mentioned in this document
+-------------------------------------------------------------------------
+CMake build script from https://github.com/HDFGroup/hdf5/tree/master/config/cmake/scripts:
     CTestScript.cmake
 
-External compression szip and zlib libraries:
-    LIBAEC.tar.gz
-    SZip.tar.gz
-    ZLib.tar.gz
+Configuration files from https://github.com/HDFGroup/hdf5/tree/master/config/cmake/scripts:
+    HDF5config.cmake
+    HDF5options.cmake
 
-External compression plugin libraries:
+External compression plugin libraries from https://github.com/HDFGroup/hdf5_plugins:
     hdf5_plugins.tar.gz
 
-Examples Source package:
-    HDF5Examples-2.0.x-Source.tar.gz
+External compression szip and zlib libraries:
+"Use ZLIB from original location"          "https://github.com/madler/zlib/releases/download/v1.2.13"
+"Use ZLIB from original compressed file"   "zlib-1.3.tar.gz"
 
-Configuration files:
-    HDF5config.cmake
-    HDF5options.cmake
+"Use LIBAEC from original location"        "https://gitlab.dkrz.de/k202009/libaec/-/archive/v1.0.6"
```

Comment:
This link requires login and 2 years old already. Is there an alternative? (E.g., https://github.com/erget/libaec/archive/refs/tags/v0.3.4.tar.gz)